server/cluster/cluster.go

```diff
@@ -1879,19 +1879,20 @@ func (c *RaftCluster) onStoreVersionChangeLocked() {
 	failpoint.Inject("versionChangeConcurrency", func() {
 		time.Sleep(500 * time.Millisecond)
 	})
+	if minVersion == nil || clusterVersion.Equal(*minVersion) {
```

Comment:
What if the cluster version is greater than minVersion?

Reply:
The case is :
1. the cluster version 5.0.5
2. put a `tikv` with v4.0.5 will be failed
3. put a `tikv` with v5.0.1 will be success and the cluster version should be v5.0.1
4. put a `tikv` with v5.0.2 should be success and the cluster version should be still v5.0.1

Reply:
So when we use API to change the cluster version, it might be changed to another value?

Reply:
The previous logic is when we set the cluster to a higher version, it won't be changed. But right now, it might be changed to another value.